examples/differential_correction_deleonberne.py

```diff
@@ -0,0 +1,224 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Tue Jul 30 10:02:48 2019
+
+@author: Wenyang Lyu and Shibabrat Naik
+
+Script to compute unstable periodic orbits at specified energies for the coupled quartic 
+Hamiltonian using differential correction
+"""
+
+import numpy as np
+import matplotlib.pyplot as plt
+from scipy.integrate import solve_ivp
+import time
+
+# This needs testing for installation 
+import sys
+import importlib
+sys.path.insert(0, './src/')
+
+import differential_correction
+importlib.reload(differential_correction)
+import differential_correction as diffcorr
+
+import deleonberne_hamiltonian
+importlib.reload(deleonberne_hamiltonian)
+import deleonberne_hamiltonian as deleonberne
+# This needs testing for installation
+
+import matplotlib as mpl

```

Comment:
The solving and plotting should be separated out into two different files, or at least clearly separated within the file.

Reply:
Solving and plotting are in separate portions: [plotting begins at the end of the solution portion](https://github.com/WyLyu/UPOsHam/blob/494cc9329400d1ff2cf9e29c69bdd6b4d7b5cba3/examples/differential_correction_deleonberne.py#L161), we are doing all the imports here.